docs/getting-started.md

```diff
@@ -9,65 +9,234 @@ sidebar_label: Getting started
+- Payroll that automatically runs by itself
+- Built-in voting on important topics
+
+#### DAO use cases: 
+
+- Part-time projects with friends or strangers
+- Future of work: people working part time on multiple things for short periods of time
+- Temporary pop-up companies
+- Companies in authoritarian jurisdictions
+- Global, distributed teams
+
+#### Trends around the world accelerating DAO adoption: 
+
+- The rise of populist authoritarians
+- The future of work (remote and distributed)
+- The growing de-platforming problem
+- The rise of decentralized finance
+- The [increasing](https://medium.com/complex-systems-channel/teams-a-manifesto-7490eab144fa) [complexity](https://necsi.edu/complexity-rising-from-human-beings-to-human-civilization-a-complexity-profile) of human civilization
+
+
+
+### Web3
+
+Unless you've been living under a rock for the last few years, you've probably come across the term web3 😋. But what does it mean exactly? And why do we care about it in the context of Aragon?
+
+Web3 is the vision of a fully decentralized web. One of the craziest things to wrap your head around is that in web3, apps don't need a central server to fetch data from!
+
+How is this possible? In a nutshell, thanks to something called peer-to-peer data architectures. The key point is that in a P2P architecture, instead of requesting data from a central server, you request it from multiple computers (peers) around you.
+
+While this is nothing new in itself -- P2P architectures have existed since the 1990’s (where they rose to fame with file sharing programs like BitTorrent and Napster) -- what's new is the addition of cryptography and economic incentives to these architectures.
 
-What you need to get started building with Aragon:
+This fusion of these seemingly disparate disciplines was the big innovation behind Bitcoin, and has since led to the emergence of a new field of research devoted to their intersection (what we now call cryptoeconomics). 
 
-- [Environment setup](#environment-setup)
-- [Quick start](#quick-start)
-- [Next steps](#next-steps)
+While we won't get into the details here,the key takeaway is that cryptoeconomics is the big unlock that has allowed us to start moving from centralized data structures (web2) to more decentralized or fully distributed data architectures (web3).
+
+![](/docs/assets/centralized-vs-decentralized-stack-2.png)
+
+> Note that there’s a spectrum from fully centralized (left) to fully decentralized (right).
+
+And while blockchains -- like Bitcoin and Ethereum -- are key to this web3 vision, it's important to note that there are other essential parts of the web3 stack that are not covered by them. For example, since blockchains are relatively expensive to store data on, it turns out that they don't make great file systems. That's why there's also a need for decentralized file systems like the [InterPlanetary File System](https://ipfs.io/) (which Aragon also makes use of).
+
+
+### Further reading
```

Comment:
Maybe include it at the bottom make sense?

Reply:
Sorry, not sure I understand. Do you mean the Further reading section?

Reply:
Yes exactly